comfyui_custom_nodes/webui_checkpoint_loader.py

```diff
@@ -23,12 +15,12 @@ def INPUT_TYPES(s):
     CATEGORY = "loaders"
 
     def load_checkpoint(self, void):
-        config = webui_proxies.get_comfy_model_config()
-        webui_proxies.raise_on_unsupported_model_type(config)
+        config = proxies.get_comfy_model_config()
+        proxies.raise_on_unsupported_model_type(config)
         return (
-            WebuiModelPatcher(WebuiModelProxy()),
-            WebuiClipWrapper(WebuiClipProxy()),
-            WebuiVaeWrapper(WebuiVaeProxy()),
+            proxies.ModelPatcher(proxies.Model()),
+            proxies.ClipWrapper(proxies.Clip()),
+            proxies.VaeWrapper(proxies.Vae()),

```

Comment:
do you think it would be a better idea to pass these values by default?

Reply:
I did consider it, but I'm not sure whether by default or not is better. Requiring to pass these values means exposing the dependencies, which should ensure ease of testing if we were to ever add unit tests for these classes.

Reply:
Also, `proxies.ModelPatcher` must receive the model as argument, since `proxies.ClipWrapper` also uses it.